changelog.asc

```diff
@@ -2,6 +2,67 @@
+** No more screenshots! Save your Statistics as a PDF
+** Audio recording and Check Pronunciation usability overhaul
+** Late night study session? Screens now refresh if you see the day rollover
+** Undo has been expanded to almost all app activities. Redo has also been added
+** New permissions screen to help new users of the app
+** Note Editor: Added ‘Cloze Deletion (same card)’ toolbar button
+** Improved support for gamepad joysticks, triggers and pedals
+** Shake is now a supported gesture 🪇🪇
+** https://ankidroid.org/#userActions[User-defined JavaScript Actions] are now supported
+** AnkiWeb rate limits for searches & downloads are replaced with a sign in screen
+** Exporting: "Include deck configs" option is enabled
+** Sync: Cancellation is supported
+** Use the Android ‘Share’ menu to create new notes from images (including Image Occlusion)
+** UX: The Deck Picker menus are easier to navigate
+** UI: Material Design has been introduced, making preferences easier to search & navigate, along with a new layout for large screens
+** UI: Better Deck Description screen (long press a deck to access)
+** UI: The Android Navigation bar now matches the active screen
+** UI: Card Info & Help screens are improved
+** ...Many more UI tweaks
+** Improved keyboard shortcuts
+*** Previewer accepts arrow keys; Ctrl+Shift+P will preview the Browser’s selection
+** Accessibility Option: minimal screen touch time to reveal answer
+* JS API (0.0.2)
+** We continue to work on new technology to allow for add-ons in AnkiDroid.
+** This is still a technology preview but is already very powerful.
+** ⚠️ BREAKING: API is now asynchronous
+** Added new API for Speech to Text
+** Handle android dialog cancellation when used as frontend for js dialogs
+* Replacements and Removals
+** Advanced Statistics have been replaced with desktop statistics screen
+** V3 scheduler replaces V2
+** Auto Advance now located in Deck Options and syncs with all Anki clients
+** Text to Speech (TTS) is now Desktop compatible!
+*** We have introduced {{tts}} and {{tts-voices:}}, which supports more TTS voices and speeds: https://docs.ankiweb.net/templates/fields.html#text-to-speech-for-individual-fields[manual]
+*** ⚠️ Our legacy TTS (<tts>) will be removed in a future version. Please migrate your card templates to the new format
+** Backup options unified with Anki Desktop
+** Night mode inversion has been removed - https://ankidroid.org/removed-features.html#_automatic_night_mode_color_inversion[more info]
+** https://ankidroid.org/removed-features.html#_custom_fonts_in_fonts[Custom fonts removed]
+** https://ankidroid.org/removed-features.html[Full information on all removed features]
+* Bug Fixes
+** Too many to mention! Hundreds of issues fixed. https://github.com/ankidroid/Anki-Android/milestone/51?closed=1[See the full changelog]
+* Release Statistics:
+** link:++https://github.com/ankidroid/Anki-Android/compare/v2.16.5...main++[958 changes from 26 contributors since 2.16]
```

Comment:
this one was really subtle! the target kept getting html-entitified on the `...`